### ISSUE TITLE
Loading pretrained weights(Epic-kitchens)

### ISSUE THREAD

**Issue by JaesungHuh** [facebookresearch/Motionformer#9]
Hi!! Thank you for releasing the code.

I tried to load the Epic-kitchens pretrained weights(ek_motionformer_224_16x4.pyth) but seems like there are some missing keys in the pytorch state_dict. 

Missing Keys:
{'blocks.9.attn.proj_kv.bias', 'blocks.7.attn.proj_q.weight', 'blocks.4.attn.proj_kv.weight', 'head0.weight', 'blocks.5.attn.proj_q.weight', 'patch_embed_3d.proj.weight', 'blocks.1.attn.proj_q.weight', 'head1.weight', 'blocks.10.attn.proj_q.weight', 'temp_embed', 'blocks.1.attn.proj_q.bias', 'blocks.10.attn.proj_kv.bias', 'blocks.1.attn.proj_kv.weight', 'pre_logits.fc.bias', 'blocks.5.attn.proj_q.bias', 'blocks.11.attn.proj_q.bias', 'blocks.2.attn.proj_q.weight', 'blocks.6.attn.proj_kv.weight', 'blocks.2.attn.proj_kv.bias', 'blocks.3.attn.proj_kv.bias', 'blocks.11.attn.proj_q.weight', 'blocks.10.attn.proj_q.bias', 'patch_embed_3d.proj.bias', 'blocks.8.attn.proj_kv.bias', 'blocks.3.attn.proj_q.bias', 'blocks.5.attn.proj_kv.weight', 'blocks.2.attn.proj_kv.weight', 'blocks.3.attn.proj_q.weight', 'blocks.9.attn.proj_kv.weight', 'blocks.9.attn.proj_q.weight', 'pre_logits.fc.weight', 'blocks.10.attn.proj_kv.weight', 'blocks.8.attn.proj_q.bias', 'blocks.5.attn.proj_kv.bias', 'blocks.0.attn.proj_kv.bias', 'blocks.4.attn.proj_kv.bias', 'blocks.0.attn.proj_q.bias', 'blocks.11.attn.proj_kv.weight', 'blocks.6.attn.proj_q.bias', 'head1.bias', 'blocks.3.attn.proj_kv.weight', 'blocks.7.attn.proj_kv.bias', 'blocks.8.attn.proj_q.weight', 'blocks.8.attn.proj_kv.weight', 'blocks.4.attn.proj_q.weight', 'blocks.6.attn.proj_kv.bias', 'blocks.7.attn.proj_q.bias', 'blocks.0.attn.proj_kv.weight', 'blocks.6.attn.proj_q.weight', 'blocks.4.attn.proj_q.bias', 'blocks.2.attn.proj_q.bias', 'blocks.1.attn.proj_kv.bias', 'blocks.9.attn.proj_q.bias', 'blocks.0.attn.proj_q.weight', 'head0.bias', 'blocks.11.attn.proj_kv.bias', 'blocks.7.attn.proj_kv.weight'}

Therefore, I could not reproduce the results with Epic-kitchens validation set. Could you please check the uploaded weights on Epic-kitchens?

**Comment by JaesungHuh** [comment]
Hi!! I debugged the code and it seems like the above missing keys are not the missing keys of the motionformer. I'll close this issue.
